.orchestra/ci/ci.sh

```diff
@@ -186,6 +186,20 @@ if test "$PUSH_CHANGES" = 1; then
             fi
         done
 
+        # Promote orchestra config next-* to *
+        cd "$ORCHESTRA_ROOT"
+        BRANCH="$(git rev-parse --abbrev-ref HEAD)"
+        if test "${BRANCH:0:5}" == "next-"; then
+            PUSH_TO="${BRANCH:5}"
+            git branch -d "$PUSH_TO" || true
+            git checkout -b "$PUSH_TO" "$BRANCH"
+            git push origin "$PUSH_TO"
+            # Also switch to it, as fix-binary-archives-symlinks
+            # reads the current orchestra config branch
+            git switch "$PUSH_TO"
+        fi
+        cd -
+
```

Comment:
Let's not duplicate. Either have a function or just add it to the loop before and then (after the loop) unconditionally do `git switch`.

Reply:
Ok